airbyte-integrations/connectors/destination-s3/build.gradle

```diff
@@ -17,6 +17,21 @@ application {
 
     // uncomment and replace to run locally
     //applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0', '--add-opens', 'java.base/java.lang=ALL-UNNAMED']
+    applicationDefaultJvmArgs = [
+            '-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0'
+//            Uncomment to enable remote profiling:
```

Comment:
curious, was this helpful?

Reply:
I figured it out locally before I ran the profiler, but the profiler helped confirm it, yes.

Specifically I could see the high # of allocations in the write method.

Reply:
great. so you managed to figure out connecting to the K8s cluster + port forwarding and all?